mmaction/models/localizers/bmn.py

```diff
@@ -134,7 +134,7 @@ def __init__(self,
         self.anchors_tmins, self.anchors_tmaxs = self._temporal_anchors(
             -0.5, 1.5)
         self.match_map = self._match_map()
-        self.bm_mask = self._get_bm_mask()
+        self.bm_mask = self._get_bm_mask().cuda()
```

Comment:
Is this a bug fix?

Reply:
Okay, could you provide insight on why this version is faster?

Reply:
Because the tensor.cuda() hostToDevice of parrots is very slow, we calculate asynchronously.